modules/lsp/src/main/scala/playground/lsp/PluginResolver.scala

```diff
@@ -78,13 +76,7 @@ object PluginResolver {
                   getClass().getClassLoader(),
                 )
 
-              ServiceLoader
-                .load(
-                  classOf[PlaygroundPlugin],
-                  classLoader,
-                )
-                .asScala
-                .toList
+              PlaygroundPlugin.getAllPlugins(classLoader)
```

Comment:
This might be useful for your tests @daddykotex @lewisjkl @yisraelU

Reply:
thanks